promenade/generator.py

```diff
@@ -28,7 +28,7 @@ def validate(self):
                 raise
 
         assert self.input_config['Cluster'].metadata['name'] \
```

Comment:
You're probably right that this should throw a more specific exception, but I wouldn't hold that against this PR, since that's a shortcut that I took previously.